src/layouts/MultiDhoLayout.vue

```diff
@@ -315,7 +379,145 @@ export default {
       } catch (error) {
 
       }
+    },
+
+    timeago (createdDate) {
+      const TODAY = new Date().toUTCString()
+      const created = new Date(createdDate).toUTCString()
+      const second = date.getDateDiff(created, TODAY, 'seconds')
+      const minute = date.getDateDiff(created, TODAY, 'minutes')
+      const hour = date.getDateDiff(created, TODAY, 'hours')
+      const day = date.getDateDiff(created, TODAY, 'days')
+      const month = date.getDateDiff(created, TODAY, 'months')
+      const year = date.getDateDiff(created, TODAY, 'years')
+      if (year > 0) {
+        if (month < 12) {
+          return `${month} month${month > 1 ? 's' : ''} ago`
+        } else {
+          return `${year} year${year > 1 ? 's' : ''} ago`
+        }
+      }
+      if (month > 0) return `${month} month${month > 1 ? 's' : ''} ago`
+      if (day > 0) return `${day} day${day > 1 ? 's' : ''} ago`
+      if (hour > 0) return `${hour} hour${hour > 1 ? 's' : ''} ago`
+      if (minute > 0) return `${minute} minute${minute > 1 ? 's' : ''} ago`
+      if (second > 0) return `${second} second${second > 1 ? 's' : ''} ago`
+
+      return ''
+    },
+
+    parsedNotification (notification) {
```

Comment:
@evgenibir this function also should be extracted from this file so it can be reused in other parts of the app as well